_format/1.1/index.md

```diff
@@ -1839,3 +1877,572 @@ An error object **MAY** have the following members:
 [error details]: #errors
 [member names]: #document-member-names
 [pagination]: #fetching-pagination
+
+## <a href="#operations" id="operations" class="headerlink"></a> Operations
+
+A server **MAY** support usage of the `operations` array to complete one or more
+fetch or mutation operations in a single request.
+
+All HTTP requests that include an `operations` array **MUST** be issued with
+`PATCH`.
+
+For the sake of clarity, requests that include operations are called
+"operations requests" while requests that perform a single fetch or mutation are
+called "singular requests".
+
+### <a href="#operation-objects" id="operation-objects" class="headerlink"></a> Operation Objects
+
+An operation object **MAY** contain any of the following members:
+
+* `op`: an operation code, expressed as a string, that indicates the type of
+  operation to perform. Valid values include:
+  - `"get"`: fetches a resource, resource identity, collection of resources, or
+     collection of resource identitities
+  - `"add"`: creates a new resource or relationship
+  - `"update"`: updates a resource or relationship
+  - `"remove"`: removes a resource or relationship
+
+* `data`: the operation's "primary data".
+
+* `included`: an array of [resource
+  objects](#document-resource-objects) that are
+  related to the operation's primary data and/or each other
+  ("included resources").
+
+* `ref`: the resource or resource collection that is the target of the
+  operation. This member serves to disambiguate the target at a particular
+  endpoint if that endpoint represents multiple resources. `ref` **MUST** be
+  expressed as an object that **MAY** contain any of the following members:
+  - `type`: resource `type`
+  - `id` or `lid`: resource identifier or local identifier
+  - `relationship`: the name of the relationship
+
+* `params`: all of the query parameters allowed for singular requests can be
+  represented in JSON form in an operation's `params` object. Possible members
+  include:
+  - `include`
+  - `sort`
+  - `filter`
+  - `page`
+  - `fields`
+
+* `links`: a [links object](#document-links) related to the operation's primary
+  data.
+
+* `meta`: a [meta object](#document-meta) that contains non-standard
+  meta-information about the operation.
+
+Different members are required for processing different types of operations, as
+described below.
+
+> Note: `op` codes are used instead of the HTTP methods prescribed for singular
+  requests. The values `"get"`, `"add"`, `"update"`, and `"remove"` align with
+  the HTTP methods `GET`, `POST`, `PATCH`, and `DELETE` used for singular
+  requests.
```

Comment:
I understand why you don't want to clash meaning with the http verbs and `op` values, but the logical alternative to me is "create", "read", "update", "delete" (CRUD). Was this considered when naming these operation types?

Reply:
Haha ... I just had this thought this morning. The original intent was to align with RFC 6902, but since we're now using `update` and not `replace`, we may just want to use the more memorable CRUD theme.